### ISSUE TITLE
Allow configuring build.rollupOptions.output.chunkFileNames in Vite config

### ISSUE THREAD

**Issue by axekan** [sveltejs/kit#13067]
### Background:

I'm using a library called [vanilla-cookieconsent](https://www.npmjs.com/package/vanilla-cookieconsent) for a cookie consent banner. When building with this library imported, there will sometimes be a chunk called `cookieconsent.esm.2bdd7b77.js`. This file gets blocked by adblockers that disable cookie notices, which causes errors that prevent SvelteKit from starting. Example error:

```
app.ce597c13.js:1 TypeError: Failed to fetch dynamically imported module: https://example.com/_app/immutable/nodes/0.ea76f7a5.js
```

![image](https://github.com/user-attachments/assets/ddc5cddf-3583-4a64-a149-327ab60d4c01)

The file can be seen blocked in devtools, and turning off my adblocker allows the page to load correctly. For reference I am using uBlock Origin with the AdGuard/uBO – Cookie Notices extension.

There is an option in Rollup to change the chunk names, and I belive changing it to something like `chunk-[hash].js` would fix this issue. However, when setting this option, you get a message saying:

```
The following Vite config options will be overridden by SvelteKit:
  - build.rollupOptions.output.chunkFileNames
```

### Describe the proposed solution

Allow this Rollup setting to be changed to avoid getting this error. I'm guessing changing it is disabled because it currently has a very specific signature, with different values based on if the build has ssr enabled or not:

https://github.com/sveltejs/kit/blob/fb146e7a19c4a125ff8a006140598f65fb8b9f68/packages/kit/src/exports/vite/index.js#L643

I'm not sure what the best way would be to let users configure this.

### Importance

nice to have

**Comment by eltigerchino** [comment]
closing as duplicate of https://github.com/sveltejs/kit/issues/1410
